scripts/release.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env zx
+import { argv, cwd } from 'process'
+
+await $`set -e`
+
+const DIR = __dirname
+
+await $`zx ${DIR}/grabthar-validate-git`
+await $`zx ${DIR}/grabthar-validate-npm`
+
+let DIST_TAG = argv.find(element => element.includes('DIST_TAG='))?.replace('DIST_TAG=', '')
+DIST_TAG ?? (DIST_TAG = 'latest')
+
+let BUMP = argv.find(element => element.includes('BUMP='))?.replace('BUMP=', '')
+BUMP ?? (BUMP = 'patch')
+
+await $`npm version ${BUMP}`
+await $`git push`
+await $`git push --tags`
+await $`zx ${DIR}/grabthar-flatten`
+
+let NPM_TOKEN = argv.find(element => element.includes('NPM_TOKEN='))?.replace('NPM_TOKEN=', '')
```

Comment:
This kept me up last night - I'm thinking this is actually passed as an environment variable, both locally and in GitHub Actions, and should be coming from `env` instead of the command line via `argv`. Looking into it some more. (cc: @gregjopa)

Reply:
`NPM_TOKEN` is passed as an `env` variable for GitHub Actions, and 2FA is used for local releases.